service/migration_manager.hh

```diff
@@ -121,7 +121,7 @@ public:
 
     future<> announce_new_keyspace(lw_shared_ptr<keyspace_metadata> ksm, api::timestamp_type timestamp);
 
-    future<> announce_column_family_update(schema_ptr cfm, bool from_thrift, std::vector<view_ptr>&& view_updates);
+    future<> announce_column_family_update(schema_ptr cfm, bool from_thrift, std::vector<view_ptr>&& view_updates, std::optional<api::timestamp_type> timestamp);
```

Comment:
Please explain (in a comment here and in the commit changelog) the role of the timestamp. It's not the schema timestamp, so what is it used for?